set.mm

```diff
@@ -96570,8 +96570,8 @@ using the Axiom of Power Sets (unlike ~ domtr ).  (Contributed by
      the Axiom of Power Sets (unlike ~ domtr ).  (Contributed by BTernaryTau,
      24-Nov-2024.) $)
   domtrfir $p |- ( ( C e. Fin /\ A ~<_ B /\ B ~<_ C ) -> A ~<_ C ) $=
```

Comment:
I wonder how the longer proof came about.  An automatic minimization seemingly finds inefficient proof encodings and replaces it with a shorter one.  The difference is not visible in the HTML proof display.

Reply:
See discussion in https://github.com/metamath/set.mm/pull/3195.

This already happened in the past. The reason these changes shorten proofs "seemingly without explanation" is because after minimizing I use `SAVE NEW_PROOF /COMPRESSED` instead of `SAVE NEW_PROOF /COMPRESSED /FAST`. The `/FAST` option avoids the application of the deduplication algorithm (briefly mentioned [here](https://github.com/metamath/set.mm/pull/3195#issuecomment-1556317763)), which can shorten proofs further. Sometimes people use the `/FAST` option after minimizing and so they miss these potential shortenings.